eol-checker/eol_checker/yaml_loader.py

```
@@ -0,0 +1,75 @@
+
+    @staticmethod
+    def download_yaml(url: str) -> dict:
+        """
+        Download the lifecycle YAML file.
+        Args:
+            url: The URL of the lifecycle YAML file.
+        Returns:
+            The lifecycle YAML file content.
+        """
+        try:
+            response = requests.get(url, timeout=30, verify=False)
```

Ruff (0.15.13)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)